public/js/sunnah.js

```diff
@@ -480,7 +480,7 @@
 	
 	if ("searchQuery" in window) {
 		$(".searchquery").val($('<textarea/>').html(searchQuery).text());
-		$(".searchquery").css('color', '#000');
+		// $(".searchquery").css('color', '#000');
```

Comment:
What did this line used to do? If we don't need it anymore please feel free to delete